src/hocon_tconf.erl

```diff
@@ -478,7 +456,9 @@ map_fields_cont([{FieldName, FieldSchema} | Fields], Conf0, Acc, Opts) ->
                 ),
                 erlang:raise(C, Err, St)
         end,
-    Conf = put_value(Opts, FieldName, unbox(Opts, FValue), Conf0),
+    Conf1 = put_value(Opts, FieldName, unbox(Opts, FValue), Conf0),
+    %% now drop all aliases
+    Conf = maps:without(Aliases, Conf1),
```

Comment:
I personally feel that I'm missing some semantics: does it mean that either the field name itself or the _first_ defined alias for a field will win in a conf file? Maybe worth to clarify it in [this comment](https://github.com/emqx/hocon/pull/235/files#diff-1f8bb16d97e0cfdeb0298b5a722454f2ad949f4491ca49f8e4971da0b33b01b8R136-R139)?